burst/src/main/java/com/squareup/burst/Burst.java

```diff
@@ -45,21 +48,33 @@
    * @throws ClassCastException If any element of {@code constructorArgs} or {@code methodArgs} is
    * not an enum value.
    */
-  public static String friendlyName(Enum<?>[] arguments) {
+  public static String friendlyName(Enum<?>[] arguments, Annotation[][] argumentAnnotations) {
```

Comment:
I think we can just add an overload which passes an empty array. This lets us make a minor release instead of a major one.